### ISSUE TITLE
Build Ambari from branch "clemlab-2.7.8.0"

### ISSUE THREAD

**Issue by Nazarii-Melnyk** [rework-space-com/ambari#1]
## Description

Ambari must be built using next command: 
```bash
mvn -B install package jdeb:jdeb "-Dmaven.clover.skip=true" "-DskipTests" "-Dstack.distribution=ODP" "-Drat.ignoreErrors=true" -Dpython.ver="python >= 2.6" -Djdk.version="1.8" -Dfindbugs.skip=true -DnewVersion=2.7.8.0 -DbuildNumber=<commit_hash> -Dviews -Pdeb "-Dodp.release.number=134"
```

Required packages: `git`, `maven`, `python2-dev`, `openjdk-8-jdk`, `bzip2`.
`JAVA_HOME` must be set.

Tasks:
 - [x] Build Ambari from branch `clemlab-2.7.8.0`

- [bower issue github](https://github.com/bower/bower/issues/2608)

**Comment by Nazarii-Melnyk** [comment]
There are some files that was removed by ClemLab, but after merge those files are present, and for now there are failures, related to this files. Build command is the same as in the description. Errors:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project ambari-logsearch-logfeeder-container-registry: Compilation failure: Compilation failure: 
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/command/DockerInspectContainerCommand.java:[23,17] package org.slf4j does not exist
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/command/DockerInspectContainerCommand.java:[24,17] package org.slf4j does not exist
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/command/DockerInspectContainerCommand.java:[36,24] cannot find symbol
[ERROR]   symbol:   class Logger
[ERROR]   location: class org.apache.ambari.logfeeder.docker.command.DockerInspectContainerCommand
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/command/DockerListContainerCommand.java:[21,17] package org.slf4j does not exist
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/command/DockerListContainerCommand.java:[22,17] package org.slf4j does not exist
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/command/DockerListContainerCommand.java:[33,24] cannot find symbol
[ERROR]   symbol:   class Logger
[ERROR]   location: class org.apache.ambari.logfeeder.docker.command.DockerListContainerCommand
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/DockerContainerRegistryMonitor.java:[21,17] package org.slf4j does not exist
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/DockerContainerRegistryMonitor.java:[22,17] package org.slf4j does not exist
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/DockerContainerRegistryMonitor.java:[30,24] cannot find symbol
[ERROR]   symbol:   class Logger
[ERROR]   location: class org.apache.ambari.logfeeder.docker.DockerContainerRegistryMonitor
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/DockerContainerRegistry.java:[25,17] package org.slf4j does not exist
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/DockerContainerRegistry.java:[26,17] package org.slf4j does not exist
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/DockerContainerRegistry.java:[44,24] cannot find symbol
[ERROR]   symbol:   class Logger
[ERROR]   location: class org.apache.ambari.logfeeder.docker.DockerContainerRegistry
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/command/DockerInspectContainerCommand.java:[36,40] cannot find symbol
[ERROR]   symbol:   variable LoggerFactory
[ERROR]   location: class org.apache.ambari.logfeeder.docker.command.DockerInspectContainerCommand
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/command/DockerListContainerCommand.java:[33,40] cannot find symbol
[ERROR]   symbol:   variable LoggerFactory
[ERROR]   location: class org.apache.ambari.logfeeder.docker.command.DockerListContainerCommand
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/DockerContainerRegistryMonitor.java:[30,40] cannot find symbol
[ERROR]   symbol:   variable LoggerFactory
[ERROR]   location: class org.apache.ambari.logfeeder.docker.DockerContainerRegistryMonitor
[ERROR] /naza_VMs/apacheAmbari/ambari-logsearch/ambari-logsearch-logfeeder-container-registry/src/main/java/org/apache/ambari/logfeeder/docker/DockerContainerRegistry.java:[44,40] cannot find symbol
[ERROR]   symbol:   variable LoggerFactory
[ERROR]   location: class org.apache.ambari.logfeeder.docker.DockerContainerRegistry
```

**Comment by Nazarii-Melnyk** [comment]
Created new branch `clemlab-2.7.8.0` based on commit `release-2.7.5`.  Branch `2.7.8.0` from Clemlab repository was merged with new branch `clemlab-2.7.8.0`, and also removed irrelevant files, which caused problems.

Ambari from branch `clemlab-2.7.8.0` was successfully built using next command:
```bash
mvn -B install package jdeb:jdeb "-Dmaven.clover.skip=true" "-DskipTests" "-Dstack.distribution=ODP" "-Drat.ignoreErrors=true" -Dpython.ver="python >= 2.6" -Djdk.version="1.8" -Dfindbugs.skip=true -DnewVersion=2.7.8.0 -DbuildNumber=c141980132d313116b5b9754917f52dd78c57d2b -Dviews -Pdeb "-Dodp.release.number=134"
```
